swift-ticketing/test/swift_ticketing/factory.clj

```diff
@@ -64,11 +76,32 @@
         seat (fn [x] {"name" (str x)})]
     {"ticket_type" (mk-ticket-type random-id)
      "description" (mk-ticket-description random-id)
-     "seat_type" "General"
+     "seat_type" "Named"
      "reservation_limit_in_seconds" (+ 50 (rand-int 200))
      "seats" (for [x (range (inc (rand-int 20)))] (seat x))
      "price" (inc (rand-int 2000))}))
 
+(defn mk-reserve-general-ticket-request [quantity ticket-type-id]
+  {"quantity" quantity
+   "ticket_type_id" ticket-type-id})
+
+(defn mk-reserve-seated-ticket-request [ticket-ids]
+  {"ticket_ids" ticket-ids})
+
+(defn mk-ticket []
+  {:ticket_id (java.util.UUID/randomUUID)
+   :ticket_name (mk-ticket-name (rand-int 1000))
+   :ticket_type_id (java.util.UUID/randomUUID)
+   :ticket_price (+ 10 (rand-int 10000))
+   :reservation_expiration_time (.plus (Instant/now)
+                                       (Duration/ofSeconds (+ 10 (rand-int 200))))
+   :ticket_status db-ticket/AVAILABLE
+   :booking_id (java.util.UUID/randomUUID)})
+
+(defn mk-tickets []
```

Comment:
This seems like a redundant function. You can remove this function and call map directly from the caller.
Checkout this blog: https://stuartsierra.com/2015/08/10/clojure-donts-redundant-map